_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+> ESMValTool can already find and read the freshly downloaded FLUXCOM data.
+>
+> > ## Answers
+> >
+> > ```bash
+> > esmvaltool run recipe_check_fluxcom.yml --log_level debug
+> > ```
+> >
+> > The ESMValTool should now find your FLUXCOM datafile, and the error
+> > message that the ESMValTool can't find the data should now not be present
+> > anymore. However, there are plenty of other error messages around.
+> > Somewhere within the many messages, you should see this:
+> >
+> > ```bash
+> > ...
+> > esmvalcore.cmor.check.CMORCheckError: There were errors in variable GPP:
+> > Variable GPP units unknown can not be converted to kg m-2 s-1
+> >  lon: standard_name should be longitude, not None
+> >  lat: standard_name should be latitude, not None
+> >  lon longitude coordinate has values < -360 degrees
+> >  lon longitude coordinate has values > 720 degrees
+> >  lat: has values < valid_min = -90.0
+> >  lat: has values > valid_max = 90.0
+> >  GPP: does not match coordinate rank
+> > in cube:
+> > gross_primary_productivity_of_carbon / (unknown) (time: 12; lat: 360; lon: 720)
+> >      Dimension coordinates:
+> >           time                                        x        -         -
+> >           lat                                         -        x         -
+> >           lon                                         -        -         x
+> >      Attributes:
+> >           created_by: Fabian Gans [fgans@bgc-jena.mpg.de], Ulrich Weber [uweber@bgc-jena.mpg...
+> >           flux: GPP
+> >           forcing: CRUNCEPv6
+> >           institution: MPI-BGC-BGI
+> >           invalid_units: gC m-2 day-1
+> >           method: Artificial Neural Networks
+> >           provided_by: Martin Jung [mjung@bgc-jena.mpg.de] on behalf of FLUXCOM team
+> >           reference: Jung et al. 2016, Nature; Tramontana et al. 2016, Biogeosciences
+> >           source_file: /mnt/lustre02/work/bd0854/b309143/Data/Tier3/FLUXCOM/OBS_FLUXCOM_reana...
+> >           temporal_resolution: monthly
+> >           title: GPP based on FLUXCOM RS+METEO with CRUNCEPv6 climate
+> >           version: v1
+> > ...
+> > ```
+> {: .solution}
+{: .challenge}
+
+The error messages that we see tell us that we do have to reformat the dataset
+slightly to have it follow the CMOR standard. It seems like the variable "gpp"
+does not have the correct unit that is given in the CMOR table where it is
+[defined](https://github.com/ESMValGroup/ESMValCore/tree/master/esmvalcore/cmor/tables/cmip6/Tables).
+It also seems like the "latitude" and "longitude" coordinates have some
+problems. So let's start writing a short python script that will fix these
+problems.
+
+The first step now is to create a file in the right folder that will contain
+the short python script. The home of all CMORizer scripts for observations
+and reanalysis datasets is
+[here](https://github.com/ESMValGroup/ESMValTool/tree/master/esmvaltool/cmorizers/obs).
```

Comment:
I suggest replacing `[here] `to `esmvaltool/cmorizers/obs` and point users to episode development-setup where they can ding these directories.

Reply:
Good point. I'll instead add a note about the development setup episode near the top of the episode.

Reply:
I added several references to the dev setup episode, and I now point to these paths explicitly